<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="258-PC" ignored="0" name="Tests" tests="2" failures="2" timestamp="2021-11-24T18:54:22 EET" time="12.784" errors="0">
  <testcase name="runScenario" time="11.284" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at stepDefinitions.LoginStepDefinitions.i_am_successfully_logged(LoginStepDefinitions.java:50)
at âœ½.I am successfully logged "false"(file:///C:/Eclipse/end-2-end-framework-with-cucumber/./src/test/java/features/Login.feature:10)
... Removed 4 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="11.304" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at academy.Listeners.onTestFailure(Listeners.java:57)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Tests -->
